corehq/apps/reports/filters/urls.py

```diff
@@ -6,13 +6,15 @@
     DeviceLogUsers,
     EmwfOptionsView,
     MobileWorkersOptionsView,
+    UserOptionsView,
     ReassignCaseOptions,
 )
 from .location import LocationGroupFilterOptions
 
 urlpatterns = [
     url(r'^emwf_options_all_users/$', EmwfOptionsView.as_view(), name='emwf_options_all_users'),
     url(r'^users_options/$', MobileWorkersOptionsView.as_view(), name=MobileWorkersOptionsView.urlname),
+    url(r'^all_users_options/$', UserOptionsView.as_view(), name=UserOptionsView.urlname),
```

Comment:
I think the URL from the earlier commit was clearer, using `users` to refer to mobile+web, then `mobile_users` for just mobile users.

Reply:
I agree but I was hesitant to change a URL. I think it should be safe since we use the URL name to get the actual URL and it's not one that anyone is going to use externally. I'll change it back.

Reply:
ca7b3532c05